ts/IterableInboxDataModel.ts

```diff
@@ -42,13 +44,20 @@ class IterableInboxDataModel {
         console.log("IterableInboxDataModel.setItemAsRead")
 
         this.inboxMessages[row].read = true
-        RNIterableAPI.setReadForMessage(this.inboxMessages[row].inAppMessage)
+        RNIterableAPI.setReadForMessage(this.inboxMessages[row].inAppMessage, true)
```

Comment:
I think we would use this.idForRow(row) for the first argument of setReadForMessage.